src/App/App.js

```diff
@@ -0,0 +1,45 @@
+import './App.scss';
+import Header from '../Header';
+import Card from '../Card';
+import cl from 'classnames';
+
+
+const loremIpsum = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras vestibulum mauris justo, non egestas sapien malesuada sit amet.';
+
+const cards = [
+  {
+    id: '0',
+    caption: 'Card one header',
+    content: loremIpsum
+  }, {
+    id: '1',
+    caption: 'Card two header',
+    content: loremIpsum
+  }, {
+    id: '2',
+    caption: 'Card three header',
+    content: loremIpsum
+  }
+].map(one =>
```

Comment:
I think it would be better to map cards in render. **Can you please imagine you receive cards from BE**.